apps/precinct-scanner/src/screens/PollWorkerScreen.test.tsx

```diff
@@ -0,0 +1,56 @@
+import { act, render, waitForElementToBeRemoved } from '@testing-library/react'
+import { electionSampleDefinition } from '@votingworks/fixtures'
+import { fakeKiosk } from '@votingworks/test-utils'
+import { NullPrinter } from '@votingworks/utils'
+import MockDate from 'mockdate'
+import React from 'react'
+import AppContext from '../contexts/AppContext'
+import PollWorkerScreen from './PollWorkerScreen'
+
+MockDate.set('2020-10-31T00:00:00.000Z')
+
+beforeEach(() => {
+  window.location.href = '/'
+  window.kiosk = fakeKiosk()
+})
+
+afterEach(() => {
+  window.kiosk = undefined
+})
+
+test('shows security code', async () => {
+  const mockTotpGet = jest.fn()
+  mockTotpGet.mockResolvedValue({
+    timestamp: '2020-10-31T01:01:01.001Z',
+    code: '123456',
+  })
+
+  window.kiosk!.totp = {
+    get: mockTotpGet,
+  }
+
+  await act(async () => {
```

Comment:
I don't think having essentially everything inside an `act` call makes sense. What's requiring this? The `render` call?